app/views/posts/_header.html.haml

```diff
@@ -2,32 +2,59 @@
 - breadcrumbs = capture do
   - if include_breadcrumbs
     = render :partial => 'posts/breadcrumbs'
-- if @parent.is_a?(Project)
+- if @parent.is_a?(Project) && @parent.is_new_project?
+  .row.project-back
+    .col-md-12
+      %a{ href: project_path( @parent ) }
+        %i.fa.fa-angle-left
+        = t( :back_to_x, noun: @parent.title )
+  .row
+    .col-md-12
+      - title = t( :x_project_journal, project: strip_tags( @parent.title ), name: t( :journal ) )
+      %h1= title
+      #post_header
+        %div
+          - if @display_project.curated_by?(current_user)
+            = link_to t(:new_post), new_project_journal_post_path(@parent.journal_slug), id: "newpostbutton",
+                class: "default button"
+        = render partial: 'search_form', locals: { parent_type: @parent.class.name, parent_id: @parent.id }
```

Comment:
I'm not a fan of the extraneous whitespace this creates below the title when the viewer can't see the "New Post" button. Could we remove the search form from the header at put it at the top of the right column instead?

Reply:
With the right column being the "archives" section? Something like:
## own
![image](https://user-images.githubusercontent.com/19367605/201531428-aaa08361-1375-468c-bc2d-67cc06537419.png)

## other user
![image](https://user-images.githubusercontent.com/19367605/201531441-5401cd07-d54b-4089-8357-e7b483ff40d9.png)

## project
![image](https://user-images.githubusercontent.com/19367605/201531591-e7df5182-f091-4a1e-86d8-6086fba74608.png)


## w/o archives:
![image](https://user-images.githubusercontent.com/19367605/201531487-3dd077a0-d1a1-4598-a446-5a188c806198.png)
### blog
![image](https://user-images.githubusercontent.com/19367605/201531554-51c028a4-d086-4390-a79a-c174c9624208.png)

Reply:
Exactly. Those screenshots look good to me.